packages/evm/src/utils/liquidity.ts

```diff
@@ -0,0 +1,28 @@
+import type { EvmResource, Environment } from '@buildwithsygma/core';
+import { Config, ResourceType } from '@buildwithsygma/core';
+
+import type { Eip1193Provider } from '../types.js';
+
+import { getEvmHandlerBalance } from './balances.js';
+
+export async function getLiquidity(
```

Comment:
@MakMuftic I would like to have a common package as well that provides liquidity information on destination chain

Currently this method is very generic and only provides token balance of a sygma handler. However, this doesn't fit the use case well. When someone would be transferring a token from one chain to another, they would like to know the liquidity on destination chain.

Currently in order to do that, a user will actually have to create a fungible transfer using `createFungibleTransfer` method from this `@buildwithsygma/evm` and then set up a destination chain provider and then call this method. Not to mention if the destination chain is substrate, this would not work because `@buildwithsygma/core` doesn't include substrate stuff at all including balance queries

While this function is good to have, it only fills half of its purpose when making a transfer, unless that logic needs to be written by end users themselves

Reply:
Honestly, I think this was mostly here to facilitate widget needs. 
I agree :100: with your thoughts on the current implementation being not as useful. 
We can throw it out from here and figure out how to implement this in separate PR tbh.

Reply:
https://github.com/sygmaprotocol/sygma-sdk/issues/383